backend/src/api_wrapper/untis_client.rs

```diff
@@ -114,7 +114,7 @@ impl UntisClient {
     }
 
     async fn get_ids(&mut self) -> Result<HashMap<String, u16>, Error> {
-        let klassen: Vec<Klasse> = self.get_klassen().await.map_err(|err| Error::UntisError(err.to_string() + " 118"))?;
+        let klassen: Vec<Klasse> = self.get_klassen().await.map_err(|err| Error::UntisError(err.to_string()))?;
```

Comment:
why

Reply:
Cause I need the error messages to be the same in all files

Reply:
ah yeah makes sense